tests/unit/plugins/module_utils/test_config_loader.py

```
@@ -0,0 +1,73 @@
+        config_file.write_text(
+            "organization: my-org\n"
+            "authentication:\n"
+            "  access-token: my-access-token\n"
+            "service:\n"
+            "  server: https://flightctl.example.com\n"
+            "  insecureSkipVerify: true\n"
+        )
+
+        loader = ConfigLoader(config_file=str(config_file))
+
+        assert loader.token == "my-access-token"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)